registry/handlers/manifests.go

```diff
@@ -481,12 +483,19 @@ func (imh *manifestHandler) DeleteManifest(w http.ResponseWriter, r *http.Reques
 		return
 	}
 
+	g := errgroup.Group{}
+	g.SetLimit(storage.DefaultConcurrencyLimit)
 	for _, tag := range referencedTags {
-		if err := tagService.Untag(imh, tag); err != nil {
-			imh.Errors = append(imh.Errors, err)
-			return
-		}
+		tag := tag
+
+		g.Go(func() error {
+			if err := tagService.Untag(imh, tag); err != nil {
+				imh.Errors = append(imh.Errors, err)
+			}
+			return nil
+		})
 	}
+	g.Wait() // imh already records all errors
```

Comment:
We need 2 things here (given that the `errgroup` goroutines return `nil`)
1. Add a comment explaining why we're not checking `error`
2. Add a marked for the linter to ignore the `errcheck` -> `// nolint:errcheck